src/dotnet-core-uninstall/Shared/Utils/Regexes.cs

```diff
@@ -41,7 +41,7 @@ internal static class Regexes
             $@"(?<{PreviewGroupName}>\-(preview{_previewVersionNumberRegex.ToString()}(\.{_buildNumberRegex.ToString()}\.\d+|\-(final|{_buildNumberRegex.ToString()}(\-\d+)?))|rc{_rcVersionNumberRegex.ToString()}\-final))");
 
         private static readonly string _sdkVersionBasicRegexFormat =
-            $@"(?<{VersionGroupName}>{_majorMinorRegex.ToString()}\.((?<{SdkMinorGroupName}>\d+)(?<{PatchGroupName}>\d{{{{2}}}})|(?<{PatchGroupName}>\d{{{{1,2}}}}))({{0}})?)";
+            $@"(?<{VersionGroupName}>{_majorMinorRegex.ToString()}\.((?<{SdkMinorGroupName}>\d+)(?<{PatchGroupName}>\d{{{{0,2}}}}))({{0}})?)";
```

Comment:
It might be better to have a majorMinorRegex static field referenced in two places.